web/src/lib/components/faces-page/suggest-merge.svelte

```diff
@@ -0,0 +1,117 @@
+    </div>
+
+    <div class="flex items-center justify-center px-2 py-4 md:h-36 md:px-4 md:py-4">
+      {#if !choosePersonToMerge}
+        <div class="flex h-28 w-28 items-center px-1 md:h-32 md:w-32 md:px-2">
+          <ImageThumbnail
+            circle
+            shadow
+            url={api.getPeopleThumbnailUrl(personMerge1.id)}
+            altText={personMerge1.name}
+            widthStyle="100%"
+          />
+        </div>
+        <div class="mx-0.5 flex md:mx-2">
+          <CircleIconButton
+            logo={Merge}
+            on:click={() => ([personMerge1, personMerge2] = [personMerge2, personMerge1])}
+          />
+        </div>
+
+        <button
+          class="flex h-28 w-28 items-center px-1 md:h-32 md:w-32 md:px-2"
+          on:click={() => {
+            if (potentialMergePeople.length > 0) {
+              choosePersonToMerge = !choosePersonToMerge;
+            }
+          }}
+        >
+          <ImageThumbnail
+            border={potentialMergePeople.length > 0 ? true : false}
+            circle
+            shadow
+            url={api.getPeopleThumbnailUrl(personMerge2.id)}
+            altText={personMerge2.name}
+            widthStyle="100%"
+          />
+        </button>
+      {:else}
+        <div class="grid w-full grid-cols-1 gap-2">
+          <div class="px-2">
+            <button on:click={() => (choosePersonToMerge = false)}> <ArrowLeft /></button>
+          </div>
+          <div class="flex items-center justify-center">
+            <div
+              class="grid grid-cols-{potentialMergePeople.length >= 2
+                ? '2'
+                : '1'} md:grid-cols-{potentialMergePeople.length}"
+            >
+              {#each potentialMergePeople as person (person.id)}
+                <div class="h-24 w-24 md:h-28 md:w-28">
+                  <button class="p-2" on:click={() => changePersonToMerge(person)}>
+                    <ImageThumbnail
+                      border={true}
+                      circle
+                      shadow
+                      url={api.getPeopleThumbnailUrl(person.id)}
+                      altText={person.name}
+                      widthStyle="100%"
+                      on:click={() => changePersonToMerge(person)}
+                    />
+                  </button>
+                </div>
+              {/each}
+            </div>
+          </div>
+        </div>
+      {/if}
+    </div>
+
+    <div class="flex px-4 md:px-8 md:pt-4">
+      <h1 class="text-xl text-gray-500 dark:text-gray-300">Are these the same face?</h1>
+    </div>
+    <div class="flex px-4 pt-2 md:px-8">
+      <p class="text-sm text-gray-500 dark:text-gray-300">They will be merged together</p>
+    </div>
+    <div class="mt-8 flex w-full gap-4 px-4 pb-4">
+      <Button color="gray" fullwidth on:click={() => dispatch('differentFaces')}>No</Button>
+      <Button fullwidth on:click={() => dispatch('mergeFaces')}>Yes</Button>
```

Comment:
Maybe `dispatch('confirm')`. It might be nice to emit the final results, instead of relying on the bound values between the parent/child components.